orthanc/orthanc-anon/plugin/pixl.py

```diff
@@ -160,37 +162,59 @@ def SendViaFTPS(resourceId: str) -> None:
     Makes a POST API call to upload the resource to an FTPS server
     using orthanc credentials as authorisation
     """
-    # Query orthanc-anon for the study
+    # Download zip archive of the DICOM resource
     query = f"{ORTHANC_URL}/studies/{resourceId}/archive"
+    fail_msg = "Could not download archive of resource '%s'"
+    response_study = _make_query(resourceId, query, fail_msg)
+
+    # get the zip content
+    zip_content = response_study.content
+    logging.info("Downloaded data for resource %s", resourceId)
+
+    upload.upload_dicom_image(zip_content, GetPatientID(resourceId))
+    logging.info("Uploaded data to FTPS for resource %s", resourceId)
+
+
+def GetPatientID(resourceId: str) -> str:
```

Comment:
why are we CamelCasing this?

Reply:
Ah yes, guess I was just following the style from the file, but makes sense to put this in snake_case and maybe make it private with as `_get_patient_id`?

Reply:
Yeah its a bit of a style call but I think anything that isn't directly used by orthanc should follow pep8